gandalf.js

```diff
@@ -1,43 +1,45 @@
-const filteredArray = array.filter(item => item !== null && item !== undefined);
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?